plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8241,47 +8241,29 @@ public void testSelectFromPrestoViewReferencingHiveTableWithTimestamps()
 
         assertThat(query(defaultSession, "SELECT ts FROM " + prestoViewNameDefault)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123'");
 
-        // TODO(https://github.com/trinodb/trino/issues/6295) Presto view schema is fixed on creation
-        // should be: assertThat(query(defaultSession, "SELECT ts  FROM hive_timestamp_nanos.tpch." + prestoViewNameDefault)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123456789'")
         assertThat(query(defaultSession, "SELECT ts  FROM hive_timestamp_nanos.tpch." + prestoViewNameDefault)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123'");
```

Comment:
The comment indicated the hive_timestamp_nanos.tpch.xxx view should return ts in nano precision (`....14.123456789`). Why isn't it the case after the change?

Reply:
The definition of the Trino view is encoded internally in the JSON saved in the metastore.

```
{"originalSql":"SELECT *\nFROM\n  test1\n","catalog":"hive","schema":"default","columns":[{"name":"ts","type":"timestamp(3)"}],"owner":"marius","runAsInvoker":false}
```

I think in this case the assumption from the comment was false.